tokio-signal/.travis.yml

```diff
@@ -0,0 +1,29 @@
+language: rust
```

Comment:
Can we take out the travis and appveyor files from here and use the root level ones?